sdks/java/expansion-service/build.gradle

```diff
@@ -42,6 +42,8 @@ dependencies {
   compile library.java.slf4j_api
   runtimeOnly library.java.slf4j_jdk14
   testCompile library.java.junit
+
+  runtime project(":sdks:java:io:snowflake")
```

Comment:
Why are we adding a dependency here ?

Reply:
I removed it and checked if everything is working, thanks for spotting this.